src/care/KeyValueSorter.h

```diff
@@ -223,8 +223,6 @@ class KeyValueSorter<T, RAJADeviceExec> {
          setFromArray(len, arr);
       }
 
-#if defined(CARE_ENABLE_IMPLICIT_CONVERSIONS)
```

Comment:
Why were these removed?

Reply:
Needed to compile both with and without implicit conversions disabled. I tried both, and I needed this  when CARE_ENABLE_IMPLICIT_CONVERSIONS was OFF also

Reply:
The actual issue is that implicit conversions need to be turned off in CHAI as well, and they haven't been. I've got a fix that will take care of this, so you can revert these changes.

Reply:
Ok, sounds good. I reverted the changes. Just make sure that things work with implicit conversions both on and off in you pull request afterwards.

Reply:
It works. I double checked.